src/main/java/com/mineboundteam/minebound/item/ToolTier.java

```diff
@@ -0,0 +1,10 @@
+package com.mineboundteam.minebound.item;
+
+import com.mineboundteam.minebound.registry.ItemRegistry;
+import net.minecraft.tags.BlockTags;
+import net.minecraft.world.item.crafting.Ingredient;
+import net.minecraftforge.common.ForgeTier;
+
+public class ToolTier {
```

Comment:
@insertStringHere Is this class technically a registry? In which case it should be under the `registry` package.

Reply:
I don't think they are no, ArmorMaterial doesn't count as a registry despite acting the same way